### DISCUSSION TITLE
Version 1.12.0.0 - personal recompile by @zer0Kerbal >>>provided 'as is' caveat emptor<<<

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/KSPChromaControl#3] [Announcements]
# Version 1.12.0.0 - provided 'as is' caveat emptor

# >>-- personal recompile by @zer0Kerbal --<<

- personal compile for KSP 1.12.1
- modernize folder structure
- backend automation

- add Version.tt

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/KSPChromaControl/releases/tag/1.12.0.0-personal'>Version 1.12.0.0 - personal recompile by @zer0Kerbal >>>provided 'as is' caveat emptor<<<</a>.</em>

**Comment by The-Beaver** [comment]
does this have to be compiled?  or just point ckan at the source zip file to install?

Thanks!

**Reply by zer0Kerbal** [reply]
> does this have to be compiled? or just point ckan at the source zip file to install?
> 
> Thanks!

hello @The-Beaver , 

This is a manual install and will be until I get a new release compiled and the SpaceDock entry done.

I just (temporarily) put the [zip 📂  💾  ](https://github.com/zer0Kerbal/KSPChromaControl/releases/download/1.12.0.0-personal/KSPChromaControl-1.12.0.0-personal.zip) into the release. 

***Caveat Emptor*** (beware) - it was just a quick recompile but it has been working fine for me.

 I will remove this archive when I release the next version (soon™)

- delete any existing version (if using CKAN - use that to remove)
- Download the archive
- scan it twice to be sure no 🐞 ( 😀 ) 
- open the archive
- drag the contents of the archive (should say by a 📁 folder named GameData) into your KSP folder - not into KSP\GameData\.
  - the contents will go into KSP\GameData\000\ which allows it to run nearly first.
- start the game normally. 


Good Luck!
